realm/realm-library/src/androidTest/kotlin/io/realm/ManagedDictionaryTesters.kt

```diff
@@ -364,6 +366,28 @@ class ManagedDictionaryTester<T : Any>(
         }
     }
 
+    override fun copyToRealmOrUpdate() {
```

Comment:
This only tests the copy part, not the update part.

Reply:
That's right 😱 You should just grab hold of the object you just created, modify the dictionary field, call `copyToRealmOrUpdate` again, and assert again. Sorry I missed this one :(

Reply:
I did miss it too